docs/api/features/pcoords.rst

```diff
@@ -5,52 +5,54 @@ Parallel Coordinates
 
-.. code:: python
+.. plot::
+
+    from yellowbrick.features import ParallelCoordinates
+    from yellowbrick.datasets import load_occupancy
 
     # Load the classification data set
-    data = load_data("occupancy")
+    X, y = load_occupancy()
 
     # Specify the features of interest and the classes of the target
     features = [
         "temperature", "relative humidity", "light", "C02", "humidity"
     ]
     classes = ["unoccupied", "occupied"]
 
-    # Extract the instances and target
-    X = data[features]
-    y = data.occupancy
-
-The visualization can be drawn with either the ``ParallelCoordinates`` visualizer or using the ``parallel_coordinates`` quick method:
-
-.. code:: python
-
-    from yellowbrick.features import ParallelCoordinates
-
     # Instantiate the visualizer
     visualizer = ParallelCoordinates(
-        classes=classes, features=features, sample=0.5, shuffle=True
+        classes=classes, features=features, sample=0.05, shuffle=True
     )
 
     # Fit and transform the data to the visualizer
     visualizer.fit_transform(X, y)
 
     # Finalize the title and axes then display the visualization
-    visualizer.poof()
-
+    visualizer.finalize()
+    visualizer.ax.grid(False)
 
-.. image:: images/parallel_coordinates.png
 
 By inspecting the visualization closely, we can see that the combination of transparency and overlap gives us the sense of groups of similar instances, sometimes referred to as "braids". If there are distinct braids of different classes, it suggests that there is enough separability that a classification algorithm might be able to discern between each class.
 
 Unfortunately, as we inspect this class, we can see that the domain of each feature may make the visualization hard to interpret. In the above visualization, the domain of the ``light`` feature is from in ``[0, 1600]``, far larger than the range of temperature in ``[50, 96]``. To solve this problem, each feature should be scaled or normalized so they are approximately in the same domain.
 
 Normalization techniques can be directly applied to the visualizer without pre-transforming the data (though you could also do this) by using the ``normalize`` parameter. Several transformers are available; try using ``minmax``, ``minabs``, ``standard``, ``l1``, or ``l2`` normalization to change perspectives in the parallel coordinates as follows:
 
-.. code:: python
+.. plot::
 
     from yellowbrick.features import ParallelCoordinates
+    from yellowbrick.datasets import load_occupancy
+
+    # Load the classification data set
+    X, y = load_occupancy()
+
+    # Specify the features of interest and the classes of the target
+    features = [
+        "temperature", "relative humidity", "light", "C02", "humidity"
+    ]
+    classes = ["unoccupied", "occupied"]
```

Comment:
This is really just something for your attention, @lwgray and you don't have to update the docs unless you think it's valid, but with the new datasets you can actually get the features and classes without specifying them manually:

```python
data = load_occupancy(return_dataset=True)

features = data.meta['features']
classes = data.meta['labels'].keys()

X, y = data.to_data() # or to_numpy or to_pandas
```

I don't know if this is helpful for the documentation since it seems a little extra to get the raw dataset object rather than directly getting the dataframe, but I thought you might be interested for your own use or tests or other documentation.

Reply:
For a new user, II think that have the features & classes spelled out would be useful

Reply:
but then again...this is an opportunity to show how the datasets module works

Reply:
Don't worry about it too much - it was more to let you know; I'd say keep the documentation the way it is.